lib/fastlane/plugin/test_center/helper/test_collector.rb

```diff
@@ -34,7 +34,7 @@ def testables
           if @only_testing
             @testables ||= only_testing_to_testables_tests.keys
           else
-            @testables ||= Plist.parse_xml(@xctestrun_path).keys
+            @testables ||= Plist.parse_xml(@xctestrun_path).keys.select {|retrievedTestable| !Fastlane::Actions::TestsFromXctestrunAction.ignoredTestables.include?(retrievedTestable) }
```

Comment:
As this line is quite long, can you open it up to be:
```
@testables ||= Plist.parse_xml(@xctestrun_path).keys.reject do |testable|
  Fastlane::Actions::TestsFromXctestrunAction.ignoredTestables.include?(testable)
end
```

Reply:
Also, can you create a new ivar `@invocation_based_tests` pulled from `options` in `initialize` so you don't have to pass it in later?